### PR TITLE
CMS: Particle Discovery Lab + NanoAOD trigger update

### PR DESCRIPTION
**Pull request by jmhogan**
Two small long-hanging docs updates:
1. Added an educational resource of mine. I chose record 49 since it's empty and adjacent to the group of educational records starting at 50, but could be changed. 
2. Updating the trigger system documentation to describe the NanoAOD branches. 

If it's better to make these changes in the new structure, I have a branch based on master with these same updates.

### CODE REVIEW

File: Dockerfile
Comment (katilp) [inline Dockerfile:1]: Is this Dockerfile part of the PR on purpose?

File: cernopendata/modules/fixtures/data/docs/cms-guide-trigger-system/cms-guide-trigger-system.md [hunk 1]
```diff
@@ -1,97 +1,128 @@
...
-Find the software and usage instructions in
+<br>
 
-- [Analysis code for extracting the trigger information from the CMS 2010 data](/record/5003)
-- [Analysis code for extracting the trigger information from the CMS 2011 data](/record/5004)
+#### <b>MiniAOD (2015 and beyond)</b>
+  
+  MiniAOD contains:
+  
+  - trigger bits accosciated to all paths
```
Comment (katilp) [inline cernopendata/modules/fixtures/data/docs/cms-guide-trigger-system/cms-guide-trigger-system.md:65]: associated

**Review by katilp — CHANGES_REQUESTED** [review]
Two minor things: A typo to fix, and is Dockerfile here on purpose.

File: Dockerfile
Comment (jmhogan) [reply Dockerfile:1]: @katilp updating the xrootd was required to build the container

**Comment by jmhogan** [comment]
ok @katilp got the fixes in and found the bad whitespace

**Comment by jmhogan** [comment]
@katilp I guess github wants you to click some button....I can't find anything on my side that I can do to say "resolved"...

**Review by katilp — APPROVED** [review]
Thanks, looks good!

**Comment by jmhogan** [comment]
@tiborsimko I'm hoping this one is ready to go. Let me know if it would be better for me to close this PR and make the same changes against master or something that uses the new file structure.

**Comment by tiborsimko** [comment]
> @tiborsimko I'm hoping this one is ready to go. Let me know if it would be better for me to close this PR and make the same changes against master or something that uses the new file structure.

Thanks, I have fixed some trailing dot in a link. It is not necessary for you to rebase against `master`, I have just made a push. I have also pinned the portal infrastructure image to a stable version as part of this merge, so that things are working well off the `master` branch. We'll be soon ready to update 'qa` globally to `master`.  I'll contact you when ready.

**Review by tiborsimko — APPROVED** [review]
Thanks, the record is now deployed on QA: https://opendata-qa.cern.ch/record/49 

(The documentation changes have not been fully deployed on QA yet due to an  [updating issue](https://github.com/cernopendata/cernopendata-portal/issues/65) that we'll hopefully get to streamline in the next few days.)

**Pull request merged**